contracts/YearnAngleVoter.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import {
+    SafeERC20,
+    SafeMath,
+    IERC20,
+    Address
+} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import {Math} from "@openzeppelin/contracts/math/Math.sol";
+
+import {IVoteEscrow} from "../interfaces/Angle/IVoteEscrow.sol";
+
+contract YearnAngleVoter {
+    using SafeERC20 for IERC20;
+    using Address for address;
+    using SafeMath for uint256;
+    
+    address constant public angle = address(0x31429d1856aD1377A8A0079410B297e1a9e214c2);
+    
+    address constant public veAngle = address(0x0C462Dbb9EC8cD1630f1728B2CFD2769d09f0dd5);
+    
+    address public governance;
+    address public strategy;
+    
+    constructor() public {
+        governance = msg.sender;
```

Comment:
We found that it's more convenient if this is by default ychad

Reply:
Got it, included